styles.css

```diff
@@ -4,8 +4,35 @@ body {
   margin: 0 auto;
 }
 
+section {
+	border-bottom: 1px solid #f3f3f3;
+}
+
+section.header {
```

Comment:
This is more specific than `.header`. Can we try to heep specificity low and use bare classes where possible, please?